pkg/deploy/lattice/rule_manager.go

```diff
@@ -360,8 +359,13 @@ func isRulesSame(modelRule *latticemodel.Rule, sdkRuleDetail *vpclattice.GetRule
 		}
 	}
 
-	// Header Match
+	// Method match
```

Comment:
Any thoughts on current structure of validations? we just keep adding them with more if blocks?
Does it make sense to convert sdkRuleDetail into latticemodel.Rule and do something like reflect.deepEqual? https://pkg.go.dev/reflect#DeepEqual